src/client/builtin_scene/instanced_mesh.hpp

```diff
@@ -512,9 +541,7 @@ namespace builtin_scene
   protected:
     mutable std::shared_mutex mutex_;
     InstanceMap idToInstanceMap_;
-    std::shared_ptr<RenderableInstancesList> opaqueInstances_;
-    std::shared_ptr<RenderableInstancesList> transparentInstances_;
-    std::map<RenderLayer, std::shared_ptr<RenderableInstancesList>> layeredInstances_;
+    std::map<RenderLayer, LayeredInstancesData> layeredInstances_;
```

Comment:
```suggestion
    std::vector<LayeredInstancesData> layeredInstances_;
```
Now this map should be converted to a vector.